scripts/twitch-proxy/update-and-reload-twitch-proxy.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+node $(pwd)/updateApiKey.js && pm2 reload twitch-proxy --update-env
```

Comment:
```suggestion
node ./updateApiKey.js && pm2 reload twitch-proxy --update-env
```
I think that's equivalent.

Reply:
Thanks for catching this. Think I was trying to get a bit too fancy here. `./...` is a lot more straightforward.